src/Base/StackWalker.cpp

```diff
@@ -810,6 +810,8 @@ class StackWalkerInternal
                     case 8:  // SymVirtual:
                         szSymType = "Virtual";
                         break;
+                    case NumSymTypes:  // Not a real option, should never hit

```

Comment:
With c++23 we could use `std::unreachable()` to mark it clearly as something that should never happen. Since we don't have luxury of so new standard maybe we could just use `assert(false)` which is already quite common in the codebase?

Reply:
Hmmm... I'm a bit unsure what `std::unreachable` does, as it seems to have the same internal behaviour as `assert(false)`. Maybe `std::raise(SIGILL)` or `std::signal(SIGABRT)`

Reply:
I implemented @tritao's suggestion.